src/Handler/UploadHandler.php

```diff
@@ -17,14 +17,15 @@
  *
  * @author Kévin Gomez <contact@kevingomez.fr>
  */
-final class UploadHandler extends AbstractHandler
+class UploadHandler extends AbstractHandler
```

Comment:
We don't want to change the code to fit testing. Please revert any change about "final", as well as any other irrelevant change (I'll mark only the first one)